packages/wrapper/test/wrapper.test.ts

```diff
@@ -230,17 +230,24 @@ describe('Test MonacoEditorLanguageClientWrapper', () => {
         await wrapper.start();
     });
 
-    test('config userConfiguration', async () => {
+    test('editorConfig semanticHighlighting.enabled workaround', async () => {
         const wrapper = new MonacoEditorLanguageClientWrapper();
         const wrapperConfig = createWrapperConfigClassicApp();
+        wrapperConfig.editorAppConfig.monacoWorkerFactory = configureMonacoWorkers;
         (wrapperConfig.editorAppConfig as EditorAppConfigClassic).editorOptions = {
             'semanticHighlighting.enabled': true,
         };
         const updatedWrapperConfig = await wrapper.init(wrapperConfig);
         expect(updatedWrapperConfig.vscodeApiConfig?.workspaceConfig?.configurationDefaults?.['editor.semanticHighlighting.enabled']).toEqual(true);
 
-        // why is this configuredByTheme?
-        const semHigh = StandaloneServices.get(IConfigurationService).getValue('editor.semanticHighlighting.enabled');
-        expect(semHigh).toEqual('configuredByTheme');
+        // await wrapper.start();
+
+        const semHigh = await new Promise<unknown>(resolve => {
+            setTimeout(() => {
+                resolve(StandaloneServices.get(IConfigurationService).getValue('editor.semanticHighlighting.enabled'));
+            }, 1000);
```

Comment:
Are you sure you want to keep that timeout? it may make the test flacky

Reply:
I reduced it to 100ms already, but it only works with a brief timeout. Let's keep for now.